code/modules/mining/lavaland/loot/tendril_loot.dm

```diff
@@ -174,9 +176,20 @@
 	if(istype(next, /turf/simulated/floor/plating/lava/smooth) || istype(current, /turf/simulated/floor/plating/lava/smooth)) //We can move from land to lava, or lava to land, but not from land to land
 		..()
 	else
-		to_chat(user, "<span class='warning'>Boats don't go on land!</span>")
+		if(last_message_time + 1 SECONDS < world.time)
+			to_chat(user, "<span class='warning'>Boats don't go on land!</span>")
 		return FALSE
 
+/obj/vehicle/lavaboat/user_buckle_mob(mob/living/M, mob/user)
+	M.weather_immunities += "lava"
```

Comment:
also, another dumb question but should this be `|=` instead of `+=`? or not because its a string

Reply:
I think both work, but you're right, it's probably better practice to use |=.